src/main/java/com/monumental/models/Tag.java

```diff
@@ -15,18 +15,14 @@
  */
 
 @Entity
-@Table(name = "tag", uniqueConstraints = {
-        @UniqueConstraint(columnNames = "id"),
-        @UniqueConstraint(columnNames = "name")
-})
 public class Tag extends Model implements Serializable {
 
     @Column(name = "name")
     @NotNull(groups = {New.class, Existing.class}, message = "Name can not be null")
     private String name;
 
     @JsonInclude(JsonInclude.Include.NON_NULL)
-    @ManyToMany(cascade = { CascadeType.ALL })
+    @ManyToMany(cascade = { CascadeType.DETACH })
```

Comment:
Don't ask me what this does or my head will explode.

Reply:
From my first Google, it seems there's not much explanation.